commons/commons-model-extended-mapping/src/main/kotlin/ebi/ac/uk/extended/mapping/to/ToSubmission.kt

```diff
@@ -18,11 +22,14 @@ fun ExtSubmission.toSimpleSubmission(): Submission = Submission(
 )
 
 private fun ExtSubmission.getSubmissionAttributes(): List<Attribute> {
-    val subAttrs = attributes.map { it.toAttribute() }.toMutableSet()
+    val subAttrs =
```

Comment:
why are we filtering it here?

Reply:
https://github.com/EBIBioStudies/biostudies-backend-services/pull/352/files/7b1cd4da93596bcedb0b382c16ffa1886d527ecf#diff-9f69c64127e65171eccdde20f28b77b1545af6262592db23b1512fade38f528cL16